app/src/main/java/com/example/nvdovin/weatherapp/ForecastActivity.java

```diff
@@ -20,22 +20,22 @@
     private RecyclerView recyclerView;
     int tempScale = 273;
```

Comment:
Use constant instead of simple variable. pay attention on http://www.oracle.com/technetwork/java/codeconventions-135099.html